wdl/tasks/NanoPlot.wdl

```diff
@@ -233,19 +237,19 @@ task NanoPlotFromBam {
 
     #########################
     RuntimeAttr default_attr = object {
-        cpu_cores:          16,
-        mem_gb:             32,
+        cpu_cores:          8,
+        mem_gb:             24,
         disk_gb:            disk_size,
         boot_disk_gb:       10,
         preemptible_tries:  0,
         max_retries:        1,
-        docker:             "quay.io/biocontainers/nanoplot:1.35.5--pyhdfd78af_0"
+        docker:             "us.gcr.io/broad-dsp-lrma/lr-nanoplot:1.40.0-1"
     }
     RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
     runtime {
         cpu:                    select_first([runtime_attr.cpu_cores,         default_attr.cpu_cores])
         memory:                 select_first([runtime_attr.mem_gb,            default_attr.mem_gb]) + " GiB"
-        disks: "local-disk " +  select_first([runtime_attr.disk_gb,           default_attr.disk_gb]) + " HDD"
+        disks: "local-disk " +  select_first([runtime_attr.disk_gb,           default_attr.disk_gb]) + " LOCAL"
```

Comment:
It's local SSDs, the type of disks that are physically attached to the CPUs.

Reply:
I should add that the reason for this is because this process is an IO bound process, i.e. the most time-consuming part is reading from the aligned BAM.